### ISSUE TITLE
Standard Motion notifications not working when Smart Detect is enabled

### ISSUE THREAD

**Issue by stoddieb** [hjdhjd/homebridge-unifi-protect#480]
**Describe The Problem:**
Describe The Problem:
I'm trying to have standard motion alerts and smart motion alerts to send me a notificaiton when either is triggered i.e I want all motion to send me a rich notification. What I have found is when I dont enable smart detect and leave it default I only get notifcations when other motion is detected (bugs, vehicle lights) and when I do enable smart detect I only get notified when a person or vehicle is detected. Is there a way to have both enabled? So I get a notification no matter what tripped it, bug, bird, person, vehicle?

I have played around with some other feature options to try and get these working another way by enabling Enable.Motion.SmartDetect.ObjectSensors and Enable.Motion.Trigger then set up a routine that works by turning on the motion tigger button when the person contact and vehicle contact is open and while the motion trigger does trigger a rich notification when I turn it on manually to test when its part of a routine like above it doesn't trigger a notification, I can see the routine working, Person Contact Opens >> Turns On the Motion Trigger button but then fails to turn on the motion sensor so I don't get the notification

My set up are all G4 cameras and I have the smart motion and standard motion zones set up correctly,

Any help would be greatly appricated

**To Reproduce:**

**Logs:**

```
Show the Homebridge logs here.
Remove any sensitive information.
```

**Homebridge Configuration:**

```json
Show the relevant portion of your homebridge config.json here, if needed.
Remove any sensitive information.
```

**Screenshots:**

**Environment:**

* **Homebridge Version:** 
* **Node Version:** 
* **Homebridge-UniFi-Protect Plugin Version:**
* **UniFi Protect Version:** 
* **UniFi Protect Controller Type:** 
* **UniFi Protect Controller Firmware Version:** 
* **Apple Device and iOS / macOS / iPadOS / tvOS Version:**
* **Operating System and OS Version:**

**Comment by hjdhjd** [comment]
This is working as intended. Smart motion detection will disable notification non-smart motion events on a given camera. It’s either/or, **not** both. But you can enable that or disable it on a per-camera basis.

Enabling motion alerts across both will result in substantial notification spam…it’s just not a wise idea, which is why I explicitly don’t allow for it. You can either have alerts for smart motion events, or traditional motion events, not both.

TL;DR: not a bug, working as intended. :smile:

**Comment by jiehanzheng** [comment]
Would it be possible to put this first part behind a feature option?
https://github.com/hjdhjd/homebridge-unifi-protect/blob/e6948831c5c206cb4ca8da847882606db8b522aa/src/protect-nvr-events.ts#L213

i.e. still allow standard motion events to be processed?  This would reach feature parity with the latest Protect app where we can individually toggle smart/standard events :)

**Comment by hjdhjd** [comment]
Nope. The issue has nothing to do with “what is possible”. The issue, as I have stated above, is that it would result in an unacceptable level of notification spam that would make notifications useless. If you would like to write a PR that deals with all of the excess motion notifications that it would generate, I’d be happy to review it and incorporate it, but I will not be doing so. I’ve already tried all this a couple of different ways and it simply makes no sense to implement, at least given the way the Protect API currently works.

You can already toggle “smart” versus “standard” motion events with the smart events feature option. What you cannot do, is have both on at the same time.

**Comment by jiehanzheng** [comment]
That's fair, thank you for the explanation (and for this awesome plugin)!
